spec/encoding_spec.rb

```diff
@@ -806,4 +806,33 @@
       expect(read).to eq(0)
     end
   end
+
+  describe ("#decode_openssh_signature") do
+    let(:signature) {
+      Base64.decode64(%Q(
+        U1NIU0lHAAAAAQAAADMAAAALc3NoLWVkMjU1MTkAAAAgoXgg2XgepEwBr3CQzkHXigy8T8
+        uvs69Dvox5fgnGgPMAAAAEZmlsZQAAAAAAAAAGc2hhNTEyAAAAUwAAAAtzc2gtZWQyNTUx
+        OQAAAECRxENUPwmbRveDvNFOc36EuyMIa6jXWbCVkEQ2dtORyFAnChmr1kHMFX4B9TQm6U
+        ssvYRRUo6ePL5DuAjLP+kD
+      ))
+    }
+
+    it "can decode a signature" do
```

Comment:
Worth adding any tests here for the various ways the signature might be malformed and should blow up?

Reply:
Good idea. Done.